js/utils/loadTemplate.js

```diff
@@ -3,7 +3,11 @@ var fs = require('fs');
 
 module.exports = function(templateFile, callback){
 
-  fs.readFile(templateFile, "utf8", function(err, ob){
+  var loc = window.location.pathname;
+  var cur_dir = loc.substring(0, loc.lastIndexOf('/'));
+
+  fs.readFile(cur_dir + '/' + templateFile, "utf8", function(err, ob){
```

Comment:
Here's some documentation on it http://shapeshed.com/writing-cross-platform-node/

It looks like path.join would put in the right separator.